### DISCUSSION TITLE
Is there a debug logging setting hidden somewhere?

### DISCUSSION THREAD

**Discussion by dsjstc** [RamonUnch/AltDrag#524] [Q&A]
AltSnap occasionally just stops working for me.  I can't reproduce it, and it sometimes takes a day or two to occur.

Is there any kind of logging I can turn on to help me figure out what's happening?

**Comment by RamonUnch** [comment]
Thee is no option for this.
You can use he build below:

[AltSnap164dbg.zip](https://github.com/RamonUnch/AltSnap/files/15098797/AltSnap164dbg.zip)

It will create an ad.log file however you will need to open it in gdb32.exe in order to see the debug info.
https://sourceforge.net/projects/tdm-gcc/files/GDB/gdb32-7.9.1-tdm-1.zip/download

then run 
`> gdb32.exe AltSnap.exe`
and then enter the `r` command to run and when you get a crash you will see the info in the console.

**Comment by dsjstc** [comment]
I ran for 2 days and was unable to reproduce my issue with the debug version.   Perhaps it's an old issue specific to 1.62; I'll submit an issue if I end up able to reproduce in 1.63.
